cmd/clusterctl/client/cluster/topology.go

```diff
@@ -39,8 +39,10 @@ import (
 
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/cluster/internal/dryrun"
```

Comment:
@ykakarap it would be great if you could take a look at the changes I made to the dry run code :slightly_smiling_face: